aikau/src/main/resources/alfresco/lists/Paginator.js

```diff
@@ -105,6 +115,18 @@ define(["dojo/_base/declare",
        * @default
        */
       currentPage: null,
+
+      /**
+       * The style of current page number. The default is "NUMBER" which indicates that just the
+       * page number should be displayed. The only alternative in the current version is
+       * "X_OF_Y" which will render the label in the form "1 of 4" (as in page 1 of 4).
+       * 
+       * @instance
+       * @type {string}
+       * @default
+       * @since 1.0.64
+       */
+      currentPageLabelStyle: "NUMBER",
```

Comment:
I thought about that, but for just two options I didn't think it would be worth it. Also, it's SFS specific at the moment